syntax = "proto2";

//Should match the states used in the state machine

enum CarState {
    IDLE = 1;
    ARM = 2;
    AUTO = 3;
    MANUAL = 4;
}